cypress/component/integration/Balance.cy.tsx

```diff
@@ -33,7 +33,11 @@ describe('Component | Integration | Balance', function () {
         cy.get('[data-testid="balance"] [data-testid="svg-icon"]').should(
           'exist',
         );
-        cy.get('[data-testid="balance"]').should('contain', '0123456.78');
+        cy.get('[data-testid="balance-amount"]').should('exist');
+        cy.get('[data-testid="balance-amount"]').should(
```

Comment:
You don't need these two here!

Reply:
Because if it contains it will mandatory exists. So, only the second is enough! :)